response_operations_ui/views/collection_exercise.py

```diff
@@ -425,7 +425,21 @@ def _upload_collection_instrument(short_name, period):
 
 
 def _unselect_collection_instrument(short_name, period):
-    success_panel = _unlink_collection_instrument()
+    success_panel = "Collection instrument removed"
```

Comment:
Because both eQ and SEFT was using the unlink functionality it was split, now SEFT doesn't use it I put it back into one